Spec/Auth.swift

```diff
@@ -2441,6 +2441,26 @@ class Auth : QuickSpec {
                     }
                 }
 
+                it("example: if is initialised with custom ttl and a TokenParams object is passed with null, then the @ttl@ used for every subsequent authorization will be null") {
+                    let options = AblyTests.commonAppSetup()
+                    options.defaultTokenParams = { $0.ttl = 0.1; return $0 }(ARTTokenParams())
+                    let rest = ARTRest(options: options)
+
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.authorise(nil, options: nil) { tokenDetails, error in
```

Comment:
@ricardopereira It should use the default, of course.

I'm not sure why the MAC signing doesn't work. Can you arrange [this](https://github.com/ably/ably-ios/blob/6a99e08d5bba78dcdcbc9242601bd0ad6bda37a7/Source/ARTTokenParams.m#L64-L65) so that it adds an empty clientId param there (`&clientId=&otherParam=...`) and try again?